docs/parsers/RSSParser.md

```diff
@@ -0,0 +1,30 @@
+---
+title: RSS Parser
+author: williamabradley
+ms.date: 01/19/2018
+description: The RSS Parser allows you to parse an RSS content String into RSS Schema.
+keywords: uwp community toolkit, uwp toolkit, microsoft community toolkit, microsoft toolkit, rss, rss parsing, parser
+---
+
+# RSS Parser
+
+The [RssParser](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.parsers.rss.rssparser) class allows you to parse a RSS content String into a RSS Schema.
+
+| Class | Purpose |
+| --- | --- |
+| **Microsoft.Toolkit.Parsers.Rss.RssParser** | Parser for Parsing RSS Strings into RSS Schema. |
+| **Microsoft.Toolkit.Parsers.Rss.RssSchema** | Schema for Parsing RSS. |
+
+## Sample Code
+
+[RSS Parser Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RssParser/RssParserPage.xaml.cs).
+
+You can see this in action in [UWP Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).
+
+## Requirements
```

Comment:
Requirements need to show the namespace and nuget package (see template: https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md)

Reply:
Done.